### ISSUE TITLE
A huge JS fps drop after opening and closing the menu a few times

### ISSUE THREAD

**Issue by mohebifar** [react-native-fellowship/react-native-side-menu#208]
It seems that there is a bug related to react-native-side-menu. When I open and close the menu for a few times, the menu animation gets slower over time until JS reaches -2 fps!

For the first and second time, the animation runs smoothly but after that it gets slower and slower.

**Comment by Kureev** [comment]
How can we reproduce it?

**Comment by mohebifar** [comment]
My code is so simple. I open and close the drawer using the gestures. 

```jsx
import React, { Component } from 'react-native';
import SideMenu from 'react-native-side-menu';
import HamburgerMenu from './HamburgerMenu';

export default class RootDrawer extends Component {
  render() {
    const menu = <HamburgerMenu/>;
    const { children } = this.props;

    return (
      <SideMenu menu={menu}>
        {children}
      </SideMenu>
    );
  }
}
```

**Comment by mohebifar** [comment]
![Bug](https://www.dropbox.com/s/3d91fy9pzlf7kyk/react-native-side-menu-bug.gif?dl=1)

**Comment by mohebifar** [comment]
Closing this as it was fixed by upgrading to RN 0.23.
